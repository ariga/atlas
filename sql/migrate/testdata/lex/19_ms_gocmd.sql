go
SELECT 1
-- comment here
GO
SELECT 2 -- another comment
GO
SELECT 3 GO
GO
SELECT 4
GOTO
SELECT 5
GO 11
SELECT 6
GO