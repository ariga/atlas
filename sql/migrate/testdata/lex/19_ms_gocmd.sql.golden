
-- end --
SELECT 1
-- comment here
-- end --
SELECT 2 -- another comment
-- end --
SELECT 3 GO
-- end --
SELECT 4
GOTO
SELECT 5
-- end --
SELECT 6